docker/caffe2/jenkins/common/install_pytorch.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+pip install numpy
+git clone https://github.com/pytorch/pytorch.git --recursive
+cd pytorch
+git checkout 7fcfed19e7c4805405f3bec311fc056803ca7afb
```

Comment:
Life will probably be better for you if this hash is an argument (somehow). It probably shouldn't be in the image name itself but maybe be fed in as an environment variable from the Jenkins script.